samigo/samigo-app/src/webapp/jsf/delivery/item/deliverMultipleChoiceSingleCorrect.jsp

```diff
@@ -89,16 +89,30 @@ should be included in file importing DeliveryMessages
   </t:dataList>
 
   </h:panelGroup>
-  <h:outputText value="<script>" escape="false" />
-  <h:outputText value="    var elBlockToFix = $('.mcscFixUpClassForSelector-#{question.itemData.itemId}');" escape="false" />
-  <h:outputText value="    $(elBlockToFix).find('div.mcscFixUpSource td').each(function(index,elLabelAndInputToMove) {" escape="false" />
-  <h:outputText value="      var contentsToMove = $(elLabelAndInputToMove).contents();" escape="false" />
-  <h:outputText value="      if (typeof contentsToMove !== 'undefined') {" escape="false" />
-  <h:outputText value="        $(elBlockToFix).find('div.mcscFixUpTarget:first').replaceWith(contentsToMove);" escape="false" />
-  <h:outputText value="      }" escape="false" />
-  <h:outputText value="    });" escape="false" />
-  <h:outputText value="    $(elBlockToFix).find('div.mcscFixUpSource').remove();" escape="false" />
-  <h:outputText value="</script>" escape="false" />
+  <script>
```

Comment:
are you reverting your previous fix?

Reply:
I'm partially reverting it, but with changes.

In my previous fix for SAK-43625 with PR #8306, I had removed the code that moved the question's answer, radio button, and icon into the radio button's label section because I thought it wasn't necessary after I made the script tag only work on that script tag's question block.  That fix made the formatting "look" correct, but I didn't realize removing that code broke being able to click the label to select the radio button.

So I'm putting that code back in, but I also had to include a new change to fix the JS error that was causing SAK-43625. 
 The new change adds another 'fake' class to use as a jquery selector, 'span.**imageClassForSelector**'

`var properImage = $(answerLabel).parent('li').find('span.imageClassForSelector')[0];`

My first fix (PR #8306) also added a bunch of <h:outputText> tags to display the <script> tag, because at the time, I thought I was hitting an error if I didn't add it to all the lines of the <script> tag (But maybe that error was from a typo or bad code?). While testing this new fix, I tried to put the <h:outputText> only around the one line that needed it, and that seemed to work.  This also makes it easier to read in Chrome web developer because my previous fix put the script tag all on one line so it was hard to debug unless you use the 'pretty print' feature in web developer.

Reply:
I'm not sure if there's an easy way to compare specific files across forks via git or github, but if you download these two files, you should be able to see the differences between this file **before** my "other" fix in #8306" and **this** PR

https://raw.githubusercontent.com/sakaiproject/sakai/135f9cbd79c05d315d886c882b1c56253d3e2913/samigo/samigo-app/src/webapp/jsf/delivery/item/deliverMultipleChoiceSingleCorrect.jsp

https://raw.githubusercontent.com/austin48/sakai/SAK-43879/samigo/samigo-app/src/webapp/jsf/delivery/item/deliverMultipleChoiceSingleCorrect.jsp